bin/deploy.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+usage() {
+    echo "$0 - Simple script to make deployments of fresh code a one command operation" >&2
+    echo "Usage: $0 [-v] [-b <branch>] [-p <path>]" >&2
+    exit 1
+}
+
+update_repo(){
+    if [[ $VERBOSE ]]; then
+        echo "Updating repository"
+    fi
+    git checkout $BRANCH
+    git pull origin $BRANCH
+}
+
+# Prevent reading virtualenv environmental variables multiple times
+activate_once(){
+    if [[ $(which python) != "${GIT_WORK_TREE}"* ]]; then
+        if [[ $VERBOSE ]]; then
+            echo "Activating virtualenv"
+        fi
+        source "${GIT_WORK_TREE}/bin/activate"
```

Comment:
Would a single repo have multiple virtual environments? 
Should [the README](https://github.com/uwcirg/true_nth_usa_portal#create-a-virtual-environment) be updated with this new path (ie <code>$ virtualenv $PROJECT_HOME**/env**</code>)?